tests/integration/conftest.py

```diff
@@ -55,3 +55,15 @@ async def override_get_db():
     # For debugging change rollback to commit.
     transaction.rollback()
     connection.close()
+
+
+def _create_ensemble(client, experiment_id, parameters=[]):
+    resp = client.post(
+        f"/experiments/{experiment_id}/ensembles", json={"parameters": parameters}
+    )
+    return resp.json()["id"]
```

Comment:
Make this into a pytest fixture rather than importing from `tests.integration.conftest` in other files:

```suggestion
@pytest.fixture
def create_ensemble(client):
    def func(experiment_id, parameters=None):
        if parameters is None:
            parameters = []
        resp = client.post_check(
            f"/experiments/{experiment_id}/ensembles", json={"parameters": parameters}
        )
        return resp.json()["id"]
    return func
```